docs/milestone-1-validation.md

```diff
@@ -0,0 +1,97 @@
+# Milestone-1 Validation
+
+* Scality Object Storage Interoperability Service (for short, **Scality OSIS**)
+* VMware Cloud Director Object Storage Extension (for short, **vCloud Director OSE**)
+
+## Prerequisites
+
+1. Run Vault & Cloud Server on a VM and expose the URL and Port
```

Comment:
Reference to the respective documentation is needed,
Also, what do we exactly mean by `expose the URL and port`?

Reply:
Vault: https://github.com/scality/Vault/blob/development/7.10/TESTING.md#how-to-run-1
Cloudserver: https://github.com/scality/cloudserver/tree/development/7.10#installation
First vault needs to be started.
Only then Cloudserver needs to be started with env variable: `S3VAULT=scality` so that it can use vault as auth backend.

We do not need to mention exact commands over in this doc.
Exact commands to rclone and run vault and then cloudserver will go in the ticket: https://scality.atlassian.net/browse/S3C-4253

Reply:
Updated the same in c32406aabea800b30939886856300f757611a70d